code/modules/mob/living/silicon/robot/robot_modules/station_vr.dm

```diff
@@ -569,6 +569,7 @@
 	can_buckle = 1
 	sprites = list(
 					"Pupdozer" = "pupdozer",
+					"V2 Engidog" = "thottbot", //Citadel RP addition
```

Comment:
Honestly, I think the borg name should get changed to be.... Less meme-istic. :V

Reply:
The borg name isn't user-facing though, as far as I know.  It's still labeled as "V2 Engidog" when choosing, and still understood as a Pupdozer model.

If it is a problem, I can look into renaming it soon.

Reply:
Not really a problem, nope. Just a purrsonal preference that it looks less.... Possibly offensive. But if there's no way anyone will see the thotbot name, it's fine. :> Thanks for your time and your lovely contribution.~

Reply:
No problem, though it is basically a direct port, if needed, if other additions such as plasteel manipulator and anything else that would be "nice to have" I can look into renaming it in the same PR as well.

Reply:
Holy shit, someone finally noticed what I named it.